src/net_processing.cpp

```diff
@@ -3245,7 +3245,7 @@ bool ProcessMessage(CNode& pfrom, const std::string& msg_type, CDataStream& vRec
         unsigned int nCount = ReadCompactSize(vRecv);
         if (nCount > MAX_HEADERS_RESULTS) {
             LOCK(cs_main);
-            Misbehaving(pfrom.GetId(), 20, strprintf("headers message size = %u", nCount));
+            Misbehaving(pfrom.GetId(), 20, strprintf("message headers size() = %u", nCount));
```

Comment:
I don't understand this change. This is a "headers" message, so the original log seems correct - that it's a headers message of size x.

Reply:
I don't disagree but opted for the smallest change. Do you think the `inv`, `getdata`, and `addr` oversized messages misbehavior logging should be changed instead? Or are they all good as-is (in which case the `headers` one remains different, and each specific one can be passed into the `test_oversized_msg()` assertion function from the caller.)

Reply:
ah ok, it's to be consistent with the other logs. That's fine.

Reply:
IMO they should all be changed to `<type> message size = %u`.  "`<type> message`" because that sounds better to my ear, and "`size`" as opposed to "`size()`" because we're talking about the value of the function, not the function itself.

That said, I'll ACK anything that's consistent among these 4 logs.

Reply:
Done.